enhancements/installer/vsphere-ipi-zonal-external-lb.md

```diff
@@ -0,0 +1,194 @@
+---
+title: vsphere-ipi-zonal-external-lb
+authors:
+  - "@jcpowermac"
+reviewers:
+  - "@rvanderp3"
+  - "@bostrt"
+approvers:
+  - "@rvanderp3"
+  - "@patrickdillon"
+api-approvers:
+  - "@JoelSpeed"
+  - "@deads2k"
+creation-date: 2022-11-02
+last-updated: 2022-11-02
+status: implementable
+see-also:
+  - "/enhancements/installer/vsphere-ipi-zonal.md"
+replaces:
+  - None
+superseded-by:
+  - None
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-9652
+
+---
+
+# Support External Load Balancer in vSphere Zonal IPI
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The goal of this enhancement is to provide the ability to install vSphere IPI zonal on separate L2 segments. As of this writing the only way to 
```

Comment:
Is this distinct from the remote workers functionality that we already have for baremetal? AFAIK that should already work for vsphere, we just haven't tested it.

https://issues.redhat.com/browse/OPNET-27

Reply:
ingress was resolved for vsphere
https://issues.redhat.com/browse/SPLAT-409

The scope is install time of a external lb without the static pods of IPI with support for api and ingress.